Zappy_IA_Report/NN/Code/Tutorial1.py

```
@@ -0,0 +1,18 @@
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)